src/test/features/claim/form/models/eligibility/eligibility.ts

```diff
@@ -14,14 +14,7 @@ describe('Eligibility', () => {
 
     it('should reject eligibility if not complete', () => {
       const errors = validator.validateSync(
-        new Eligibility(
-          YesNoOption.YES,
-          YesNoOption.YES,
-          ClaimValue.UNDER_10000,
-          YesNoOption.YES,
-          YesNoOption.NO,
-          undefined
-        )
+        new Eligibility(YesNoOption.YES, YesNoOption.YES, ClaimValue.UNDER_10000, YesNoOption.NO, YesNoOption.YES, undefined)
```

Comment:
Is that still working? You made constructor accept 8 args while you provide only 6.

Reply:
well it's not complete so it will still pass, the values are optional in the constructor so it will compile,
it's not as complete as before and it's harder to read, 
please revert to old style and add the extra options
REQ

Reply:
Done